.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "tools/external"]
+	path = tools/external
```

Comment:
nit: I don't think we should scrub the name of the submodule.  `tools/external/snap-testing-tools` or `tools/snapd-testing-tools` are more clear than `tools/external`

Reply:
I agree! Updated